scripts/tests/lnv2-inter-federation.sh

```diff
@@ -8,4 +8,5 @@ source scripts/_common.sh
 build_workspace
 add_target_dir_to_path
 
-tests inter-federation
\ No newline at end of file
+# TODO: fix flakiness and reactivate or remove test completely
+# tests inter-federation
```

Comment:
Flakiness should be better now after https://github.com/fedimint/fedimint/pull/6217. Please revert

Reply:
Done